### ISSUE TITLE
Script bz.py failing on 10.13 (High Sierra)

### ISSUE THREAD

**Issue by Blatman** [codezri/buildzri#3]
OS: 10.13.6 High Sierra
Python 3.9.12 (brew installed as python3).

Trying to build neutralino 4.5.0 binary for mac 10.13

Run ./scripts/bz.py in source folder and get error -

Get the BuildZri ascii splash  and version line OK then -

Compiling Neutralinojs...
/bin/sh: -c: line 0: syntax error near unexpected token `('
/bin/sh: -c: line 0: `c++ --std=c++17 -I . -I lib -I lib/asio/include helpers.cpp chrome.cpp resources.cpp extensions_loader.cpp main.cpp settings.cpp auth/permission.cpp auth/authbasic.cpp server/router.cpp server/neuserver.cpp api/app/app.cpp api/filesystem/filesystem.cpp api/storage/storage.cpp api/extensions/extensions.cpp api/window/window.cpp api/os/os.cpp api/events/events.cpp api/computer/computer.cpp api/debug/debug.cpp api/clipboard/clipboard.cpp lib/tinyprocess/process.cpp lib/easylogging/easylogging++.cc lib/platformfolders/platform_folders.cpp lib/clip/clip.cpp lib/clip/image.cpp lib/tinyprocess/process_unix.cpp lib/clip/clip_osx.mm -DNL_VERSION=\"4.5.0\" -DNL_COMMIT=\"fatal: not a git repository (or any of the parent directories): .git\" -DELPP_NO_DEFAULT_LOG_FILE=1 -DASIO_STANDALONE -DWEBVIEW_COCOA=1 -DTRAY_APPKIT=1 -DOBJC_OLD_DISPATCH_PROTOTYPES=1 -arch x86_64 -Wno-deprecated-declarations -framework WebKit -framework Cocoa -Os -o ./bin/neutralino-mac_x64 '
ERR: Unable to compile Neutralinojs

Appears to be an issue with a GIT repository.

Need to get this going on 10.13 since change from using .sh scripts to build neutralino binaries in 4.5.0

Thanks

**Comment by shalithasuranga** [comment]
Hey.. thanks for posting this well-explained issue. We've recently added the `BZ_COMMIT` global variable to BuildZri. It uses `git` commands to get the latest commit ([implementation](https://github.com/neutralinojs/neutralinojs/blob/e07da7aaabac90bb6848be2c115e84e052b92218/scripts/bz.py#L38))

Seems like you've downloaded the repo as a zip file? Will you be able to download it with `git clone`?

**Comment by Blatman** [comment]
Yup OK that was the trick - did a git clone and the compile worked fine.

Probably be a good idea to doc that in the server repository in a similar fashion to the reference in the client repo since the server repo just points to building apps with the CLI and omits mention of building the server binary.

Thanks

**Comment by shalithasuranga** [comment]
Hey.. @Blatman  Thanks for checking.. mm not sure I fully understood your idea -- do you mean it's better to document this git scenario in the contribution guide? or?

**Comment by Blatman** [comment]
@shalithasuranga - just mean that the Client repository gives details on how to build the client lib (Developers FAQ) from that repo using git clone but the Server repo doesn't do the equivalent for the server binary it just refers to the CLI for building apps and of course the CLI only uses copied binaries it doesn't build them. It is useful info for Devs who can not use the release binaries which are made for later OS releases and the CLI uses these release binaries also.
So basically the server repo just needs a Dev FAQ guide similar to the client repo that states a binary can be built using git clone and the bz.py script with python3 a pre-requisite.

Cheers.
